core/modules/user/user.install

```diff
@@ -1344,6 +1352,999 @@ function user_update_1023() {
   }
 }
 
+/**
+ * Add a "changed" column to the "users" table.
+ */
+function user_update_1024() {
+  // On older databases originating from Drupal 5, the "changed" column was
+  // renamed to "access" in system_update_136(), and the "changed" index on the
+  // "access" column may still persist. That column index needs to be removed
+  // before adding the new "changed" column.
+  // See: https://api.drupal.org/api/drupal/modules%21system%21system.install/function/system_update_136/5.x
+  if (db_index_exists('users', 'changed')) {
+    if (!db_index_exists('users', 'access')) {
+      db_add_index('users', 'access', array('access'));
+    }
+    db_drop_index('users', 'changed');
+  }
+
+  $spec = array(
+    'type' => 'int',
+    'not null' => TRUE,
+    'default' => 0,
+    'description' => 'Timestamp for when user was changed.',
+  );
+  $keys = array(
+    'indexes' => array(
+      'changed' => array('changed'),
+    ),
+  );
+
+  // In some cases sites have added the changed field themselves e.g. via a
+  // contrib or custom module. Ensure the field uses core's new schema.
+  if (db_field_exists('users', 'changed')) {
+    db_change_field('users', 'changed', 'changed', $spec, $keys);
+  }
+  else {
+    db_add_field('users', 'changed', $spec, $keys);
+    // Set the initial value for existing users.
+    db_update('users')
+      ->expression('changed', 'created')
+      ->execute();
+  }
+}
+
+/**
+ * Add a "changed" column to the user_admin View.
+ */
+function user_update_1025() {
+  // See if the content admin view has been modified on this site.
+  $config = config('views.view.user_admin');
+
+  // Only update if view isn't overridden (VIEWS_STORAGE_OVERRIDE).
+  if ($config->get('storage') == 2) {
+    return;
+  }
```

Comment:
Why not check for `VIEWS_STORAGE_DEFAULT` here instead (`$config->get('storage') == 4`), and then execute the rest of that code in that case?

Reply:
You probably meant to check for `VIEWS_STORAGE_OVERRIDE`?

Reply:
We can use constants in the update process, thus the 2.

@klonos I'm not sure what benefit your suggestion provides.

This update hook is very similar to what is in node.install.